config/secrets.rb

```diff
@@ -0,0 +1,14 @@
+default: &default
+  secret_key_base: <%= ENV["SECRET_KEY_BASE"] %>
+
+  development:
+      <<: *default
+
```

Comment:
Is this indentation correct? Would have thought it should be outside of `default` key.